internal/cmd/login.go

```diff
@@ -526,34 +597,71 @@ func promptLoginOptions(cli *CLI, client *api.Client) (loginMethod loginMethod,
 	}
 }
 
-// Error out if it fails TLS verification and user does not want to connect.
-func promptSkipTLSVerify(cli *CLI) error {
-	// Although the same error, format is a little different for interactive/non-interactive.
-	fmt.Fprintf(os.Stderr, "  %s\n", ErrTLSNotVerified.Error())
-	confirmPrompt := &survey.Confirm{
-		Message: "Are you sure you want to continue?",
+func promptVerifyTLSCert(cli *CLI, cert *x509.Certificate) error {
+	// TODO: improve this message
+	// TODO: use color/bold to highlight important parts
+	// TODO: test format with golden
+	fmt.Fprintf(cli.Stderr, `
+The certificate presented by the server is not trusted by your operating system. It
+could not be automatically verified.
+
+Certificate
+
+Subject: %[1]s
```

Comment:
This is awesome. I wonder if we can trim this down a bit – e.g. do we need validity, SANs, etc?

Reply:
Ok, I removed the SAN fields. We definitely don't need those since they will be verified automatically by the client.

I left the validity times, but I added a human readable version first which hopefully makes it read better.